pallets/lottery/types.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file looks abnormal.
Follow: https://polkadot.js.org/docs/api/start/types.extend#definition-clashes